Tools/wasm/emscripten/web_example/index.html

```diff
@@ -201,8 +243,21 @@
                                     this.input + this.writeLine("\n"),
                                 );
                                 this.input = "";
+                                this.cursorPosition = 0;
                                 this.activeInput = false;
                                 break;
+                            case "\x03": // CTRL+C
+                                this.input = "";
+                                this.xterm.write("\n");
+                                this.xterm.write(
+                                    "\x1b[" + (this.cursorPosition + 4) + "D",
+                                );
```

Comment:
Native console also writes out `KeyboardInterrupt` when you press `CTRL+C`, why not do that here too?